tests/tests.toml

```diff
@@ -370,6 +370,16 @@ filename = "../integration_tests/print_01.py"
 c = true
 cpp = true
 
+[[test]]
+filename = "../integration_tests/generics_01.py"
+asr = true
+llvm = true
+
+[[test]]
+filename = "../integration_tests/generics_array_01.py"
+asr = true
+llvm = true
```

Comment:
Same here, let's add an integration test and remove llvm here.

Keep testing asr, that is good.